container/kvm/initialisation.go

```diff
@@ -95,24 +89,36 @@ func getRequiredPackages(a string) []string {
 // createPool creates the libvirt storage pool directory. runCmd and chownFunc
```

Comment:
Comment needs updating for the new method name.

Reply:
Updated